pkg/compute/rust.go

```diff
@@ -206,15 +195,15 @@ func (r Rust) Verify(out io.Writer) error {
 	// user should be the person making the local configuration change.
 	//
 	/* #nosec */
-	cmd = exec.Command("rustup", "target", "list", "--installed", "--toolchain", r.config.File.Language.Rust.ToolchainVersion)
+	cmd = exec.Command("rustup", "target", "list", "--installed", "--toolchain", r.toolchain.String())
```

Comment:
Maybe we need to consider updating [this guide](https://developer.fastly.com/learning/compute/#install-local-dependencies) to explicitly state that you need `rustup` – installing Rust and Cargo manually will cause issues here.

> Compiling Rust applications for Compute@Edge requires Rust 1.49.0 and the wasm32-wasi toolchain. Install Rust and its dependencies so you can begin building Compute@Edge services:

Reply:
👍🏻 yup, looks like that should be updated

Reply:
Ticket opened ✨